invokeai/app/api/routers/models.py

```diff
@@ -90,23 +49,28 @@ async def list_models(
     return models
 
 @models_router.post(
-    "/",
+    "/{base_model}/{model_type}/{model_name}",
     operation_id="update_model",
     responses={200: {"status": "success"}},
 )
 async def update_model(
-    model_request: CreateModelRequest
+        base_model: BaseModelType = Path(default='sd-1', description="Base model"),
+        model_type: ModelType = Path(default='main', description="The type of model"),
+        model_name: str = Path(default=None, description="model name"),
+        info: Union[tuple(MODEL_CONFIGS)]  = Body(description="Model configuration"),
 ) -> CreateModelResponse:
```

Comment:
If this is to update a model, the response model should be like `UpdateModelResponse`

Reply:
Done.